spec/features/card_spec.rb

```diff
@@ -0,0 +1,14 @@
+# encoding: utf-8
+require 'rails_helper'
+require 'capybara/rspec'
+
+describe "visiting /check page" do
+  let(:card) { create(:card) }
+  it "put wrong answer" do
+    visit '/check'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.